packages/patternfly-4/react-core/src/components/Popover/Popover.tsx

```diff
@@ -24,84 +27,84 @@ export const PopoverPosition = {
   right: 'right'
 };
 
-const propTypes = {
+export interface PopoverProps {
   /** Popover position */
-  position: PropTypes.oneOf(Object.keys(PopoverPosition).map(key => PopoverPosition[key])),
+  position?: BasicPlacement;
   /** If true, tries to keep the popover in view by flipping it if necessary */
-  enableFlip: PropTypes.bool,
+  enableFlip?: boolean;
   /** Popover additional class */
-  className: PropTypes.string,
+  className?: string;
   /** The reference element to which the popover is relatively placed to */
-  children: PropTypes.element.isRequired,
+  children: ReactElement<any>;
```

Comment:
We've settled on `React.ReactNode`. Use that instead.

Reply:
Hi, converted other uses of `ReactElement<any>` to `ReactNode`, however `children` is passed to third-party `Tippy` component which has it's own typedef defined and expects a `ReactElement`.

Reply:
Excellent reason to use ReactElement, all good!